content/docs/integrations/slack.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Slack
+description: Learn how to integrate your Apify actors with Slack. This article guides you from installation through to automating your whole workflow in Slack.
+menuWeight: 11.2
+paths:
+    - integrations/slack
+---
+
+# Slack integration
+
+[Slack](https://slack.com/) allows you to install various services in your workspace in order to automate and centralize tasks. Apify is one of these services, and it allows you to run your Apify actors, be notified about their run status, and get your results, all without opening your browser.
+
+## Get started
+
+To use the Apify integration for Slack, you will need:
+
+- An [Apify account](https://console.apify.com/).
+- A Slack account (and workspace).
+
+## [](#step-one) Step 1: Create a task in Apify Console
```

Comment:
I didn't get why the task is created first, but is it for the purpose of showing how to run it later directly from slack?

Reply:
Yep, I think that should be ok.